### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Gormitiz** [ethereum-optimism/developers#673] [Feedback]
### Feedback

I don’t know if this the right place to say this but i would like to know if i could get acess to my eth through you, I recently sent some crypto from binance to blockchain.com through optimism, and they say they can’t do anything about this
Thank you

**Comment by imtipi** [comment]
blockchain.com has private key of your deposit address,they can help you but they might think this not worth to do it.
https://support.blockchain.com/hc/en-us/articles/8219099905436-Lost-Crypto-Recovery
and seems they don't support recovery in op mainnet

**Reply by Gormitiz** [reply]
So its not possible to get this any other way? Only if blockchain start supporting optimism i will get my ETH?

**Reply by hash1go** [reply]
Whether you can recover assets sent to a CEX on an unsupported network depends on the CEX's support. At minimum, the chain operator does not have the ability to recover them.

**Comment by mdqst** [comment]
> I don’t know if this the right place to say this but i would like to know if i could get acess to my eth through you, I recently sent some crypto from binance to blockchain.com through optimism, and they say they can’t do anything about this
> Thank you

I can offer guidance on what steps you might take:

1. **Confirm the Transaction Details**  
   - Verify the transaction hash on the Optimism blockchain explorer to ensure it was successful.
   - Check that you sent the funds to the correct address.

2. **Contact Blockchain.com Support**  
   - If you sent funds to a Blockchain.com wallet, but they don't support Optimism, the funds might not appear in your wallet. Reach out to Blockchain.com’s support team and provide the transaction details, including the hash and the destination address.

3. **Contact Binance Support**  
   - If you suspect an error on Binance’s end, provide them with the transaction hash and ask for clarification.

4. **Consider a Recovery Service**  
   - If the funds were sent to an unsupported network or incompatible wallet, recovery might be possible depending on the platform’s policies. Some services specialize in recovering misplaced cryptocurrency.

5. **Prevention for Future Transactions**  
   - Always double-check the destination wallet’s network compatibility before transferring funds. Sending assets over an unsupported network often leads to complications like this.

I hope this helps, and I wish you the best of luck in resolving the issue!
